File: src/integration-tests/results/rs/pets.rs
use serde::{Serialize, Deserialize};
use std::collections::HashMap;
extern crate serde_json;

pub type Man = serde_json::Value;
pub type Bear = serde_json::Value;
pub type Pig = serde_json::Value;
#[derive(Serialize, Deserialize)]
pub enum OneOfBearManPigRIVnq1Ij {
    Man,
    Bear,
    Pig
}
/// Pets
///
/// a bunch of pets
///
pub type Pets = Vec<OneOfBearManPigRIVnq1Ij>;

File: src/integration-tests/results/py/pets.py
from typing import Any, NewType

Man = NewType("Man", Any)
from typing import Any, NewType

Bear = NewType("Bear", Any)
from typing import Any, NewType

Pig = NewType("Pig", Any)
from typing import NewType, Union

OneOfBearManPigRIVnq1Ij = NewType("OneOfBearManPigRIVnq1Ij", Union[Man, Bear, Pig])
from typing import List, NewType
"""a bunch of pets
"""
Pets = NewType("Pets", List[OneOfBearManPigRIVnq1Ij])
